src/webui/www/private/rename_files.html

```diff
@@ -56,7 +56,7 @@
                 });
 
                 const checkboxTH = tableHeaders[0];
-                checkboxHeader.injectInside(checkboxTH);
+                checkboxTH.appendChild(checkboxHeader);
```

Comment:
```suggestion
                checkboxTH.append(checkboxHeader);
```
Maybe you can use `Element.append()` globally instead?
IMO it should be preferred over `Node.appendChild()` whenever feasible. Because `appendChild` is a bit too low-level (operating at the document Node level rather than the usual Element level).